com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -709,9 +712,9 @@ define([
         var presentJobContract = !(typeof jobRole.job_contract_id === 'undefined');
 
         if (invalidDate && presentJobContract && bothJustSet) {
-          vm.onContractEdited(null, role_id).then(function () {
+          vm.onContractEdited(null, jobRole.job_contract_id).then(function () {
```

Comment:
Question: why is this `jobRole.job_contract_id` and not `role_id`, as it was before? Why did it change?

Reply:
Actually, `vm.onContractEdited` accepts job contract id not  the job role. So, previously passed the wrong param as `role_id`.